metadata-ingestion/src/datahub/ingestion/source/delta_lake/delta_lake_utils.py

```diff
@@ -33,12 +26,8 @@ def read_delta_table(
             storage_options=opts,
             without_files=not delta_lake_config.require_files,
         )
-    except Exception as e:
-        if (DELTALAKE_VERSION_GTE_0_10_0 and isinstance(e, TableNotFoundError)) or (
-            not DELTALAKE_VERSION_GTE_0_10_0
-            and isinstance(e, PyDeltaTableError)
-            and "Not a Delta table" in str(e)
-        ):
+    except TableNotFoundError as e:
+        if "Not a Delta table" not in str(e):
```

Comment:
I don't think this will work?
Before: PyDeltaTableError("Not a Delta table") => will suppress exception
After: PyDeltaTableError("Not a Delta table") => will raise exception